Source/BoogieDriver/BoogieDriver.cs

```diff
@@ -11,11 +11,10 @@ public static int Main(string[] args)
     {
       Contract.Requires(cce.NonNullElements(args));
 
-      var options = new CommandLineOptions
+      var options = new CommandLineOptions(new ConsolePrinter())
```

Comment:
Are we piggybacking on the command line options object to pass the printer around for simplicity because the options object is already being passed around?

Reply:
I'd say yes but I'd also say I'd like for `CommandLineOptions` to really be `Options`, not pertaining to the command line per se, so we can use that for any configuration we have for 'headless Boogie'